api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -447,6 +447,10 @@ message Admin {
   // Indicates whether :ref:`global_downstream_max_connections <config_overload_manager_limiting_connections>`
   // should apply to the admin interface or not.
   bool ignore_global_conn_limit = 6;
+
+  // Optional routes list for admin which are accessible
+  // if not specified, the allow list check will be skippped
+  repeated string allow_listed_routes = 7;
```

Comment:
may be a StringMatcher rather then simple string.

Reply:
Done